test/e2e/apps/rc.go

```diff
@@ -84,6 +97,175 @@ var _ = SIGDescribe("ReplicationController", func() {
 	framework.ConformanceIt("should release no longer matching pods", func() {
 		testRCReleaseControlledNotMatching(f)
 	})
+
+	ginkgo.It("should test the lifecycle of a ReplicationController", func() {
+		testRcName := "rc-test"
+		testRcNamespace := ns
+		testRcInitialReplicaCount := int32(1)
+		testRcMaxReplicaCount := int32(2)
+		rcResource := schema.GroupVersionResource{Group: "", Version: "v1", Resource: "replicationcontrollers"}
+
+		rcTest := v1.ReplicationController{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:   testRcName,
+				Labels: map[string]string{"test-rc-static": "true"},
+			},
+			Spec: v1.ReplicationControllerSpec{
+				Replicas: &testRcInitialReplicaCount,
+				Selector: map[string]string{"test-rc-static": "true"},
+				Template: &v1.PodTemplateSpec{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:   testRcName,
+						Labels: map[string]string{"test-rc-static": "true"},
+					},
+					Spec: v1.PodSpec{
+						Containers: []v1.Container{{
+							Name:  testRcName,
+							Image: "nginx",
+						}},
+					},
+				},
+			},
+		}
+
+		ginkgo.By("creating a ReplicationController")
+		// Create a ReplicationController
+		_, err := f.ClientSet.CoreV1().ReplicationControllers(testRcNamespace).Create(context.TODO(), &rcTest, metav1.CreateOptions{})
+		framework.ExpectNoError(err, "Failed to create ReplicationController")
+
+		// setup a watch for the RC
+		rcWatchTimeoutSeconds := int64(60)
```

Comment:
Seems low given this has to schedule and spin up pods and maybe pull images, and scale too. There are some existing "pod start timeout" values listed, we should be greater than that, perhaps by a lot. The thing to realize is that this is for the entire life of the watch. That is, I believe the watch will timeout after 60s whether there is traffic on it or not (not 100% sure but I think so).

Reply:
This seems likely the issue. 
How would I determine which is a good length of time to timeout the watch?

Reply:
In 957ab9afafd, I've updated it to 180s